rpcs3/Emu/Cell/PPUModule.cpp

```diff
@@ -782,7 +782,14 @@ static void ppu_check_patch_spu_images(const ppu_segment& seg)
 			}
 		}
 
-		ppu_loader.success("SPU executable hash: %s (<- %u)%s", hash, applied.size(), dump);
+		if (!applied.empty())
```

Comment:
Can just reverse the statement and save us the 'not' everywhere

Reply:
Okay.